src/mlpack/core/tree/hollow_ball_bound.hpp

```diff
@@ -35,6 +36,8 @@ class HollowBallBound
   math::RangeType<ElemType> radii;
   //! The center of the ball bound.
   arma::Col<ElemType> center;
+  //! The center of the hollow.
+  arma::Col<ElemType> hollowCenter;
```

Comment:
I don't understand; what is the purpose of the hollow center? When would it be different than the normal center? Maybe I have overlooked something.